internal/sirius/case_test.go

```diff
@@ -125,3 +125,61 @@ func TestCaseNoPayments(t *testing.T) {
 		})
 	}
 }
+
+func TestCaseWithFeeReduction(t *testing.T) {
```

Comment:
Im not sure I understand the purpose of this test?

Reply:
I need a new case with an id different to the payments case (`800`) or no payment case (`801`) because those cases are set up to return a payment or no payment